active/progress/ProgressRing.md

```diff
@@ -146,25 +45,47 @@ isn't the type's default (for example an int-typed property that doesn't default
-| Value | double | Gets or sets the current setting of the range control, which may be coerced. | 
-| ShowPaused | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate a Paused state to the user.|
-| ShowError | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate an Error state to the user. |
-| Maximum | double | Defaults to 100. Gets or sets the highest possible Value of the range element. |
-| Minimum | double | Defaults to 0. Gets or sets the Minimum possible Value of the range element. |
-| ValueChanged | event | Event that occurs when the range value changes. |
+ProgressRing can now be used in scenarios where progress is determinate or indeterminates. With this additional capability, the determinate ProgressRing does not represent a "hung" state where the user cannot interact with the app. Previously, guidance recommended that ProgressRing only be used when the user cannot continue to interact with the app, but this is no longer the only use case and ProgressRing can be used in scenarios where user interaction can continue while the ring is spinning.
+
+Below are properties being added to ProgressRing. The IsIndeterminate property of ProgressRing will default to True for backwards compatability.
+
+|Name | Type | Description | Default | 
+|:--|:-:|:-:|:-:|
+| IsIndeterminate | boolean | Gets or sets a value that indicates whether the progress ring reports generic progress with a repeating pattern (indeterminate progress) or reports progress based on the Value property (determinate progress) | True | 
+| IsActive | boolean | Gets or sets a value that indicates whether the progress ring is being displayed to the end user. If IsActive is false, the ProgressRing is not shown, but space is reserved for it in the UI layout. To not reserve space for it, set its Visibility property to Collapsed. | True |
+| Value | double | Gets or sets the current setting of the range control, which may be coerced. The value ranges from 0 to 100. | 0 | 
+| DeterminateSource | AnimatedVisualSource | Gets or sets the Lottie animation file used for the determinate state of the ring. | N/A | 
+| IndeterminateSource | AnimatedVisualSource | Gets or sets the Lottie animation file used for the indeterminate state of the ring. | N/A | 
+| ValueChanged | event | Event that occurs when the range value changes. | N/A | 
 
 # API Details
 <!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+```xaml
+Unsealed runtimeclass ProgressRing : Microsoft.UI.Xaml.Controls.Control.RangeBase
+{
+    ProgressRing();
+ 
+    Boolean IsActive;
+    Boolean IsIndeterminate; 
+    Double Value; 
+
+    IAnimatedVisualSource DeterminateSource{ get; set; };
+    IAnimatedVisualSource IndeterminateSource{ get; set; };
+
+    ProgressRingTemplateSettings TemplateSettings{ get; };
```

Comment:
Some of our controls (CalendarView, NavigationView, TeachingTip, to name a few) expose a DependencyProperty named TemplateSettingsProperty. Some controls do not.  Is that needed here? @mikehillberg

Reply:
I remember a similar conversation with @ranjeshj  where the TemplateSettings implementation across different controls is inconsistent 🤔

Reply:
I believe the prescribed pattern for TemplateSettings is a read only property (and not to expose a public dependency property).

Reply:
If it's not a DependencyProperty then the {Binding} shouldn't work. Is there a DependencyProperty getting registered that's just not getting exposed? How does [PersonPicture](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.PersonPicture) work?

Reply:
Yes. It gets registered but not exposed.

Reply:
Ok so I can remove this from the API details but do I need to add somewhere else in the doc that this control has template settings?

Reply:
I think we need this API.. I was wondering if we also needed a TemplateSettingsProperty property.

Reply:
@ranjeshj @MikeHillberg thoughts on this?

Reply:
Yes, we need the TemplateSettings property with just the getter. What we can remove is the TemplateSettingsProperty dependency property which can be internal.

Reply:
I already have the TemplateSettings property as a getter so I think we are good here! Does anyone have anymore feedback?